SharedUtils/src/main/java/com/braintreepayments/api/HTTPClientException.java

```diff
@@ -3,9 +3,9 @@
 /**
  * Exception thrown when an HTTP request fails.
  */
-public class HttpClientException extends Exception {
+public class HTTPClientException extends Exception {
```

Comment:
Changing this public exception class name would be a breaking change right? Might have to leave this as is 😞

Reply:
Ah true 🤦  I'll revert this one and maybe put up a next major version comment.